src/Handlebars/Loader/FilesystemLoader.php

```diff
@@ -62,18 +63,49 @@ class FilesystemLoader implements Loader
      */
     public function __construct($baseDirs, array $options = array())
     {
+        $this->setBaseDir($baseDirs);
+        $this->handleOptions($options);
+    }
+
+    /**
```

Comment:
Cleary didn't add this method... I think git lost its mind when I cut code and put it below as new functions